### ISSUE TITLE
Cannot read property 'getter' of null

### ISSUE THREAD

**Issue by cdnrg-rt** [fastify/fastify-multipart#190]
## 🐛 Bug Report

TypeError: Cannot read property 'getter' of null
 Updated to Fastify 3.9 and fastify-multipart 3.3.1
Two different servers both having the same error message

TypeError: Cannot read property 'getter' of null
    at checkReferenceType (/home/ryan/Code/fastify/node_modules/fastify/lib/decorate.js:39:45)
    at Object.decorateRequest (/home/ryan/Code/fastify/node_modules/fastify/lib/decorate.js:83:3)
    at fastifyMultipart (/home/ryan/Code/fastify/node_modules/fastify-multipart/index.js:177:11)

## To Reproduce

I have tried removing the fastify-multpart registration and server starts correctly
simply adding it back causes the error. I have removed node_modules and tried a fresh install and also get the same error.

```js
require('dotenv').config();
const appPath = `./${process.env.APP_PATH}`;
const app = require('fastify')();
const helmet = require('fastify-helmet');
app.register(require('fastify-multipart'), {
  attachFieldsToBody: true,
  sharedSchemaId: '#MultipartFileType'
})
```

## Expected behavior
Should register multipart plugin and server should start

```js
Fastify server should start
```

## Your Environment

- *node version*: 14
- *fastify version*: >=3.9.0
- *os*: Linux

**Comment by mcollina** [comment]
It's a regression in fastify, we'll fix it in 3.9.1 shortly.

fastify/fastify#2718

**Comment by cdnrg-rt** [comment]
Thanks for the quick response. Appreciate it.
